From d6d48ddd4c3a5ef04749475ed75d26330a3e0dc6 Mon Sep 17 00:00:00 2001
From: Joe Orton <jorton@apache.org>
Date: Tue, 19 Sep 2017 16:38:40 +0000
Subject: [PATCH] * test/testlock.c (test_timeoutcond): Increase fudge factor
 in   checking timeout precision; 100ms failed regularly in Fedora builds,  
 500ms has Never Failed(TM).

git-svn-id: https://svn.apache.org/repos/asf/apr/apr/trunk@1808910 13f79535-47bb-0310-9956-ffa450edef68
---
 test/testlock.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/testlock.c b/test/testlock.c
index 40da791b7..cd48768ca 100644
--- a/test/testlock.c
+++ b/test/testlock.c
@@ -334,7 +334,7 @@ static void test_timeoutcond(abts_case *tc, void *data)
             continue;
         }
         ABTS_INT_EQUAL(tc, 1, APR_STATUS_IS_TIMEUP(s));
-        ABTS_ASSERT(tc, "Timer returned too late", end - begin - timeout < 100000);
+        ABTS_ASSERT(tc, "Timer returned too late", end - begin - timeout < 500000);
         break;
     }
     ABTS_ASSERT(tc, "Too many retries", i < MAX_RETRY);
-- 
2.19.1

